src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -695,6 +695,13 @@ private function registerDebugConfiguration(array $config, ContainerBuilder $con
      */
     private function registerRouterConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
+        if (!$this->isConfigEnabled($container, $config)) {
+            $container->removeDefinition('console.command.router_debug');
+            $container->removeDefinition('console.command.router_match');
```

Comment:
https://travis-ci.org/symfony/symfony/jobs/261303308#L2572 should be updated to FQCN?